tsconfig.json

```diff
@@ -5,7 +5,27 @@
         "outDir": "lib",
         "rootDir": "src",
         "strict": true,
-        "declaration": true
+        "declaration": true,
```

Comment:
By changing the tsconfig, you only fix `typescript-styled-plugin` inside this project. Instead we want to change the defaults that this plugin itself contributes

Reply:
@mjbvz I completely understood!! :)  Thank you so much for the detailed review. Then, can I modify the readme? Or maybe I don't need any action.

I'll revert the commit!

Reply:
I think we can either document this or update the defaults to include these properties